libuserhelper.cpp

```diff
@@ -13,6 +13,8 @@
 
 #include <libuser/user.h>
 
+#define LOG_FUNC_INFO << "on function" << __FUNCTION__
```

Comment:
This feels like unnecessary. You can already do all kind of tricks with the logging categories with QT_MESSAGE_PATTERN environment variable and function name should be printed already. You'll find default under /var/lib/environment.

What you could do is that you put the updated QT_MESSAGE_PATTERN to the env upon daemon start. Some good tips can be found for instance from wobog blog: https://woboq.com/blog/nice-debug-output-with-qt.html

Reply:
Removed that commit. Thanks!